js/app/historyStore.js

```diff
@@ -262,7 +262,7 @@ const HistoryStore = new Lang.Class({
     // the same after a link click, factoring out this common function. When we
     // diverge in future interactions we should revisit this decomposition.
     show_ekn_id: function (ekn_id) {
-        Engine.get_default().get_object_by_id(ekn_id, null, (engine, task) => {
+        Engine.get_default().get_object_by_id(ekn_id, null, null, (engine, task) => {
```

Comment:
the `null, null` is a bit too microsoft-y for my taste. you can easily do `let cancellable = null;` as i commonly do to document this param, but app_id being null falling back to the default app id just seems.... weird? i really dont know what to do though